code/style.css

```diff
@@ -1,4 +1,258 @@
+  font-size: 20px;
+  margin-left: 15px;
+}
+
+.big-news-text {
+  font-family: "IBM Plex Sans", sans-serif;
+  font-size: 18px;
+  text-align: center;
+}
+
+/*HORISONTAL RULE*/
+.hr1 {
+  border-width: 8px 0px 0px 0px;
+  border-color: #99ddcc;
+  border-radius: 10%;
+  margin-bottom: 20px;
+}
+
+/* NEWS SCROLL SECTION*/
+
+.news-scroll {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-content: space-around;
+  margin-bottom: 10px;
+}
+
+.news-card {
+  /* display: flex; */
+  color: darkcyan;
+  background-color: #bad7df;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  padding: 20px 30px;
+  border: 3px solid darkmagenta;
+  margin: 20px;
+  width: 80%;
+  text-align: center;
+  box-shadow: 0px 8px 3px -8px #111, 
+                0px 6px 0px -3px pink,
+                0px 11px 3px -8px #111, 
+                0px 12px 0px -6px pink, 
+                0px 21px 8px -15px pink
+}
+
+.news-card:hover {
+  border-color: #99ddcc;
+  border-width: 10px;
+  border-style: solid;
+}
+
+.news-card p {
+  /* max-width: 50%; */
+  font-size: 16px;
+}
+
+.news-card img {
+  width: 100%;
+}
+
+/*FOOTER*/
+
+.footer {
+  width: 100%;
+  background-color: #bad7df;
+  color: white;
+  text-align: center;
+}
+
+
+/* MEDIA SECTION */
+
+/*TABLET*/
+@media (min-width: 768px) {
+
+  .header-section {
+    max-width: 768px;
+    margin: auto;
+  }
+
+  .navbar {
+    display: flex;
+  }
+
+  .nav-menu {
+    display: flex;
+    justify-content: flex-end;
+    max-width: 40%;
+  }
+
+.titlelogo {
+  width: 100%;
+}
+ 
+.main-heading {
+  font-size: 35px;
+}
+
+  .nav-menu p {
+    margin-right: 15px;
+    height: 20px;
+    padding-top: 30px;
+  }
+
+  .big-news {
+    display: block;
+    max-width: 100%;
+  }
+  .big-news-text {
+    max-width: 95%;
+    margin-top: 20px;
+  }
+
+  .main-image {
+    max-width: 95%;
+    margin-top 20px;
+  }
+   .news-scroll {
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    width: 100%;
+    margin: auto;
+  }
+  .news-card {
+    width: 250px;
+  }
+}
+
+/*DESKTOP*/
+@media (min-width: 992px) {
+
+  .header-section {
+    max-width: 992px;
+    margin: auto;
+  }
+
+  .nav-menu {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .nav-menu p {
+    width: auto;
+  }
+
+  .main-heading {
+    font-size: 45x;
+  }
+
+  .big-news {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    max-width: 992px;
+    margin: auto;
+  }
+
+  .main-image {
+    max-width: 50%;
+    padding-top: 60px;
+    }
+
+   .big-news h2 {
+    font-size: 35px;
+  }
+
+  .big-news-text {
+    max-width: 45%;
+    margin: 0px 0px 0px 20px;
+  }
+
+  .big-news p {
+    margin-top: 30px;
+  }
+
+  .news-scroll {
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: nowrap;
+    max-width: 992px;
+    margin: auto;
+  }
+
+  .news-card {
+    justify-content: space-around;
+    text-align: center;
+  }
```

Comment:
If you got rid of the side margin in mobile, you can add it here again in the desktop media query:
`margin: 20px;`